lib/ui/bottom_nav_bar.dart

```diff
@@ -46,37 +46,33 @@ class _BottomNavBarState extends State<BottomNavBar> {
         },
         items: [
           BottomNavigationBarItem(
-              backgroundColor: Colors.blue,
-              title: Text(
-                "Events",
-              ),
-              icon: Icon(
-                Icons.event,
-              )),
+            backgroundColor: Colors.blue,
+            label: "Events",
```

Comment:
Can you remove all the redundant changes added by superfluous commas (`,`)? 

It's best not to change the formatting of files for larger projects so that every PR only changes what's absolutely necessary.

Reply:
I am not very clear about this. What exactly do you mean?

Reply:
Do you want me to remove or add commas?

Reply:
Remove the ones you added in this PR.

You can simply revert this file `bottom_nav_bar.dart`

Reply:
Also, please share some screenshots.

Reply:
I see the `title` field has been deprecated, lets keep these changes then.